README.md

```diff
@@ -151,6 +151,8 @@ app.set('view engine', 'jade');
 
 ## Plugins
 
+You can find all of the PostHTML plugins in the [searchable catalog](http://maltsev.github.io/posthtml-plugins/).
```

Comment:
Although a few plugins are missing now (because they aren't published in the npm registry; I'll contact the authors to fix it), I'd like to collect all plugins in the catalog. That was the main point of creating the catalog :-)